### ISSUE TITLE
Skipping over CSS

### ISSUE THREAD

**Issue by indolering** [ashaffer/gulp-inline#19]
```
<template>
  <core-selection id="selection" multi="{{multi}}" on-core-select="{{selectedHandler}}"></core-selection>
  <link rel="stylesheet" type="text/css" href="../bower_components/bower_components/core-list/core-list.css">
  <div id="viewport" class="core-list-viewport"><content></content></div>
</template>
```

It's skipping over the above CSS stylesheet.  Any guesses as to why?

**Comment by ashaffer** [comment]
I just merged in something that might fix this ( #20 ).  Can you try reinstalling and let me know if the issue is resolved?

**Comment by indolering** [comment]
Sorry, I blew that config up.  I'll reopen if I run into it again.
